newIDE/app/src/MainFrame/EditorContainers/StartPage/Carousel.js

```diff
@@ -0,0 +1,338 @@
+import GDevelopThemeContext from '../../../UI/Theme/ThemeContext';
+import { useResponsiveWindowWidth } from '../../../UI/Reponsive/ResponsiveWindowMeasurer';
+
+export type YoutubeThumbnail = {|
+  link: string,
+|};
+
+export type ShowcaseThumbnail = {|
+  title: string,
+  imageSource: string,
+|};
+
+type GameTemplateThumbnail = {|
+  title: string,
+  imageSource: string,
+|};
+
+type SkeletonThumbnail = {|
+  skeleton: boolean,
+  key: string,
+|};
+
+type Props = {|
+  title: string,
+  items: ?(
+    | Array<YoutubeThumbnail>
+    | Array<ShowcaseThumbnail>
+    | Array<GameTemplateThumbnail>
+  ),
+  displayTitleSkeleton?: boolean,
+|};
+
+const referenceSizesByWindowSize = {
+  imageHeight: {
+    small: 130,
+    medium: 150,
+    large: 180,
+  },
+  arrowWidth: {
+    small: 18,
+    medium: 24,
+    large: 30,
+  },
+};
+
+const cellSpacing = 12;
+const titleHeight = 30;
+const spacerSize = 4;
+
+const styles = {
+  itemTitle: {
+    textOverflow: 'ellipsis',
+    overflow: 'hidden',
+    overflowWrap: 'break-word',
+    whiteSpace: 'nowrap',
+  },
+  gridList: { position: 'relative' },
+  image: {
+    objectFit: 'cover',
+  },
+  itemTitleContainer: { height: titleHeight },
+  leftArrowContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    marginTop: '32px',
+  },
+  rightArrowContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'center',
+    alignItems: 'center',
+    position: 'sticky',
+    right: '0',
+    top: '6px',
+    backgroundColor: 'rgba(100,100,100,0.8)',
+  },
+};
+
+const Carousel = ({ title, items, displayTitleSkeleton = true }: Props) => {
+  const theme = React.useContext(GDevelopThemeContext);
+  const [
+    shouldDisplayLeftArrow,
+    setShouldDisplayLeftArrow,
+  ] = React.useState<boolean>(false);
+  const scrollView = React.useRef<?HTMLUListElement>(null);
+  const itemsToDisplay =
+    items ||
+    Array(3)
+      .fill({
+        skeleton: true,
+      })
+      .map((item, index) => ({ ...item, key: `key${index}` }));
+
+  const windowSize = useResponsiveWindowWidth();
+  const imageHeight = referenceSizesByWindowSize.imageHeight[windowSize];
+  const arrowWidth = referenceSizesByWindowSize.arrowWidth[windowSize];
+  const cellWidth = (16 / 9) * imageHeight;
+  const widthUnit = cellWidth + cellSpacing;
+
+  let cellHeight = imageHeight;
+  if (
+    (!items && displayTitleSkeleton) ||
+    itemsToDisplay.some(item => Object.keys(item).includes('title'))
+  )
+    cellHeight += titleHeight + spacerSize;
+
+  const renderThumbnail = React.useCallback(
+    (
+      item:
+        | YoutubeThumbnail
+        | GameTemplateThumbnail
+        | ShowcaseThumbnail
+        | SkeletonThumbnail
+    ): ?React.Node => {
+      if (!item.link && !item.imageSource && !item.skeleton) return null;
+      if (item.link) {
+        return (
+          <a href={item.link} target="_blank">
+            <img
```

Comment:
Maybe we can add a "alt" and "title" with some content (a string) coming from the item (typically a title)